pkg/activitypub/store/storeutil/storeutil.go

```diff
@@ -7,6 +7,8 @@ SPDX-License-Identifier: Apache-2.0
 package storeutil
```

Comment:
Not something for this PR... but if you were going for naming parity with the storage stuff in aries-framework-go, then this package should be `storageutil` instead of `storeutil`. This is just a minor thing I noticed - it's not very important

Reply:
(I only mention this because I noticed that `spi` for instance had the same name)

Reply:
Although I see that `memstore` is named differently as well, so maybe the names weren't meant to be identical. If so, please just disregard this :)

Reply:
Yeah, I'd have to rename the parent directory too - /store -> /storage. Maybe some other time :)